### PR TITLE
Add Makerfabs ESP32-S2 Parallel TFT board

### PR DESCRIPTION
**Pull request by deshipu**
The basic settings and pins are fine, I'm still working on getting the
display to work, I will update it when I get the right init sequence.

More information:
https://www.makerfabs.com/wiki/index.php?title=ESP32-S2_Parallel_TFT_with_Touch(3.5%27%27)

### CODE REVIEW

**Comment by tannewt** [comment]
Any update on this?

File: ports/espressif/boards/makerfabs_esp32s2_parallel_tft/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+USB_MANUFACTURER = "Espressif"
+
+IDF_TARGET = esp32s2
+
+INTERNAL_FLASH_FILESYSTEM = 1
+LONGINT_IMPL = MPZ
+
+# The default queue depth of 16 overflows on release builds,
+# so increase it to 32.
+CFLAGS += -DCFG_TUD_TASK_QUEUE_SZ=32
```
Comment (MicroDev1) [inline ports/espressif/boards/makerfabs_esp32s2_parallel_tft/mpconfigboard.mk:13]:
```suggestion
```
These have been moved to common `mpconfigport.mk` and can be removed.

**Review by MicroDev1 — CHANGES_REQUESTED** [review]

File: ports/espressif/boards/makerfabs_esp32s2_parallel_tft/mpconfigboard.h [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#define MICROPY_HW_MCU_NAME         "ESP32S2"
+
+#define MICROPY_HW_NEOPIXEL         (&pin_GPIO18)
+
+#define CIRCUITPY_BOOT_BUTTON       (&pin_GPIO0)
+
+#define DEFAULT_UART_BUS_RX         (&pin_GPIO44)
+#define DEFAULT_UART_BUS_TX         (&pin_GPIO43)
+
+#define BOARD_USER_SAFE_MODE_ACTION translate("pressing boot button at start up.\n")
```
Comment (MicroDev1) [inline ports/espressif/boards/makerfabs_esp32s2_parallel_tft/mpconfigboard.h:39]:
```suggestion
```
This message is now the default and can be removed.

**Comment by deshipu** [comment]
> Any update on this?

@tannewt No updates. The display doesn't work correctly either in the 8bit or ther 16bit mode, and I couldn't get it working with the current libraries. There is some work to be done with the parallelio before this is working, but I don't currently have the cycles to work on it.

**Comment by dhalbert** [comment]
We discussed closing draft issues that are stuck in the CircuitPython meeting on 2022-11-28 in In the Weeds.

Closing for now. Can be reopened. Tracking in issue #7266, which will remain open.

**Pull request closed without merging**